### ISSUE TITLE
Gantt height

### ISSUE THREAD

**Issue by bazzacad** [frappe/gantt#91]
I have my gantt defined as so:

```
<div style="border-style: solid; border-color: red; height: 600px;" id="gantt"></div>

	var tasks = [
		{
		    id: 'Task 1',
		    name: 'Redesign website',
		    start: '2018-7-1',
		    end: '2018-7-30',
		    progress: 20,
		},
		{
			id: 'Task 2',
			name: 'website 2',
			start: '2018-7-05',
			end: '2018-7-15',
			progress: 80,
			dependencies: 'Task 1'
		},
		{
			id: 'Task 3',
			name: 'website 3',
			start: '2018-7-10',
			end: '2018-7-20',
			progress: 50,
			dependencies: 'Task 2'
		},
		{
			id: 'Task 4',
			name: 'website 4',
			start: '2018-7-10',
			end: '2018-7-20',
			progress: 50,
			dependencies: 'Task 2'
		},
		{
			id: 'Task 5',
			name: 'website 5',
			start: '2018-7-10',
			end: '2018-7-20',
			progress: 50,
			dependencies: 'Task 2'
		},  
	]
	var gantt = new Gantt("#gantt", tasks);
```

But no matter how many tasks I add, the horiz. scroll bar on the bottom is always too far away from the last task row. And if I change the height of the div, the scroll bar doesn't match the height, just the div gets bigger. What's going on?

![2018-07-05 14_41_45-tse tasks](https://user-images.githubusercontent.com/23222121/42349452-a5542c9a-8061-11e8-8829-143926dd0c38.png)

**Comment by chenhongen** [comment]
Hi,you can see code in frappe-gantt.js line 1268 like this:
` 
$.attr(this.$svg, {
   height: grid_height + this.options.padding + 100,
    width: '100%'
});
`
you can delete  "+ 100"

**Comment by ywlearn** [comment]
> Hi,you can see code in frappe-gantt.js line 1268 like this:
> ` $.attr(this.$svg, { height: grid_height + this.options.padding + 100, width: '100%' });`
> you can delete "+ 100"

I delete "+ 100",but it doesn't work

**Comment by core44** [comment]
> Hi,you can see code in frappe-gantt.js line 1268 like this:
> ` $.attr(this.$svg, { height: grid_height + this.options.padding + 100, width: '100%' });`
> you can delete "+ 100"

Not sure editing frappe-gantt.js is wise advice. Any changes will be lost if you update. Might be better to set the svg height attribute after you init the gantt....

var gantt = new Gantt("#gantt", data);
var new_height = gantt.$svg.getAttribute('height') - 100;
gantt.$svg.setAttribute('height', new_height);

**Comment by safwansamsudeen** [comment]
@bazzacad what might be the use case of this?

It isn't too hard to implement.

**Comment by safwansamsudeen** [comment]
After #393 is merged, the Gantt height will be a constant 300 pixels. This is useful for a fixed header (as asked in #115).

Closing this for now.
